### DISCUSSION TITLE
April 5th, 2024 release

### DISCUSSION THREAD

**Discussion by tphakala** [tphakala/birdnet-go#100] [Announcements]
### 🚀 Features

- MQTT publishing support, contribution by @janvrska
- Location filter threshold is now configurable value under BirdNET node

### 🏗️ Building

- *(deps)* Bump gorm.io/gorm from 1.25.8 to 1.25.9

<hr /><em>This discussion was created from the release <a href='https://github.com/tphakala/birdnet-go/releases/tag/v0.5.1'>April 5th, 2024 release</a>.</em>

**Comment by unosonic** [comment]
"file" command has disappeared from the linux amd64 build? or am I overlooking something?
```
Available Commands:
  authors     Print authors of BirdNET-Go
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  license     Print the license of BirdNET-Go
  realtime    Analyze audio in realtime mode
```

**Reply by tphakala** [reply]
It was temporarily removed due to code refactoring, it was restored by commit b34c44543ad43a9633f09e1fb2ca8d8206eff385 which was done after v0.5.1 release was issued. File analysis is not very good though as it reads whole file into memory before analysis which makes large file analysis quite slow and dependent on system RAM size.

**Reply by unosonic** [reply]
I guess realtime is the most useful option but sometimes not easily possible, eg. see https://github.com/tphakala/birdnet-go/discussions/32
if wavs are small, it's quite fast.
